src/services/EthService/network/providerHandler.ts

```diff
@@ -163,8 +174,16 @@ export class ProviderHandler {
    */
   public async getTokenInformation(tokenAddress: TAddress): Promise<TokenInformation | undefined> {
     return this.injectClient(async (client) => {
+      let _tokenAddress = tokenAddress;
+      /**
+       * ethersjs doesn't support EIP-1191 checksum validation.
+       * This will allow to {Add Custom Token} form fields to be filled.
+       */
+      if (client.network.chainId === 30 || client.network.chainId === 31) {
+        _tokenAddress = tokenAddress.toLowerCase() as TAddress;
```

Comment:
Heh, I don't like this. Have to think more about whether there is a better solution. 🤔  cc @Mrtenz

Reply:
The better solution would be to have support for it in Ethers.js. Maybe we should open an issue about it.

Reply:
Let me revert the changes in this file then and discuss it with Ethers.js project.

Reply:
Another temporary solution (until Ethers.js supports EIP-1191 validation) might be the case that if a user provides a valid EIP-1191 checksummed address, then convert it with `ethUtils.toChecksumAddress`, otherwise invalidate the input.